Storybook does not work on current main branch

### Expected Behaviour
`yarn storybook` should work

### Actual Behaviour
`yarn storybook` command is not working after 64d4f3ca33baf47d441878971a3b30ee317dacc0. I tried running storybook on previous commit (b78ec02c445ab39c5dd5ebb79eb1b8265f471090) and that works fine.

### Reproduce Scenario (including but not limited to)
Local development

#### Steps to Reproduce
```
git clone https://github.com/adobe/spectrum-web-components.git
cd spectrum-web-components
yarn
yarn storybook
```

#### Platform and Version
* Windows 10 (I tried WSL too)

#### Logs taken while reproducing problem
**CLI:**

```bash
[4:25:26 PM] Starting compilation in watch mode...

error Command failed with exit code 1.
$ /mnt/c/Projects/spectrum2/node_modules/.bin/tsc --build ./packages/action-bar/tsconfig.json ./packages/accordion/tsconfig.json ./packages/action-button/tsconfig.json ./packages/action-group/tsconfig.json ./packages/action-menu/tsconfig.json ./packages/avatar/tsconfig.json ./packages/asset/tsconfig.json ./packages/banner/tsconfig.json ./packages/base/tsconfig.json ./packages/button/tsconfig.json ./packages/bundle/tsconfig.json ./packages/button-group/tsconfig.json ./packages/checkbox/tsconfig.json ./packages/card/tsconfig.json ./packages/color-area/tsconfig.json ./packages/coachmark/tsconfig.json ./packages/color-handle/tsconfig.json ./packages/color-loupe/tsconfig.json ./packages/color-slider/tsconfig.json ./packages/color-wheel/tsconfig.json ./packages/dialog/tsconfig.json ./packages/divider/tsconfig.json ./packages/field-group/tsconfig.json ./packages/dropzone/tsconfig.json ./packages/icon/tsconfig.json ./packages/field-label/tsconfig.json ./packages/icons/tsconfig.json ./packages/icons-ui/tsconfig.json ./packages/icons-workflow/tsconfig.json ./packages/iconset/tsconfig.json ./packages/illustrated-message/tsconfig.json ./packages/link/tsconfig.json ./packages/menu/tsconfig.json ./packages/meter/tsconfig.json ./packages/modal/tsconfig.json ./packages/number-field/tsconfig.json ./packages/overlay/tsconfig.json ./packages/picker/tsconfig.json ./packages/progress-bar/tsconfig.json ./packages/popover/tsconfig.json ./packages/progress-circle/tsconfig.json ./packages/quick-actions/tsconfig.json ./packages/radio/tsconfig.json ./packages/search/tsconfig.json ./packages/shared/tsconfig.json ./packages/sidenav/tsconfig.json ./packages/slider/tsconfig.json ./packages/split-button/tsconfig.json ./packages/split-view/tsconfig.json ./packages/status-light/tsconfig.json ./packages/switch/tsconfig.json ./packages/styles/tsconfig.json ./packages/tabs/tsconfig.json ./packages/tags/tsconfig.json ./packages/textfield/tsconfig.json ./packages/theme/tsconfig.json ./packages/thumbnail/tsconfig.json ./packages/toast/tsconfig.json ./packages/tooltip/tsconfig.json ./packages/top-nav/tsconfig.json ./packages/tray/tsconfig.json ./packages/underlay/tsconfig.json -w --clean
error TS6370: Options 'clean' and 'watch' cannot be combined.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

Listening to CSS...
[4:25:28 PM] Found 0 errors. Watching for file changes.
```

**Browser:**

![image](https://user-images.githubusercontent.com/1174072/122778281-2234e880-d2ad-11eb-8615-2d7c7f0f9550.png)

All components are listed here with 404.

--- a/tasks/build-packages.js
+++ b/tasks/build-packages.js
@@ -16,11 +16,11 @@ import globby from 'globby';
 
 const buildPackage = (paths, watch) => {
     const args = ['tsc', '--build', ...paths];
-    if (watch) {
-        args.push('-w');
-    }
     try {
         execSync(`yarn ${args.join(' ')} --clean`);
+        if (watch) {
+            args.push('-w');
+        }
         execSync(`yarn ${args.join(' ')}`);
     } catch (error) {
         console.log(error.stdout.toString('utf8'));

